fetcher.go

```diff
@@ -13,8 +14,11 @@ import (
 //Fetcher is an interface for managing work messages
 type Fetcher interface {
 	Queue() string
+	InProgressQueue() string
 	Fetch()
 	Acknowledge(*Msg)
+	Active(bool)
```

Comment:
Can we please rename this to `SetActive`, I think it would be a better name.

Reply:
Renamed in fixup: 8ae85e1c6e9047312b29e23e4256a882d7272e08